kitsune/wiki/jinja2/wiki/includes/survey_form.html

```diff
@@ -1,4 +1,4 @@
-<div class="survey-container" x-data="surveyForm" data-survey-type="{{ survey_type }}">
+<div class="survey-container" x-data="surveyForm" data-survey-type="{{ survey_type }}" data-page-url="{{ page_url }}">
```

Comment:
I don't think this is needed, right? Maybe a leftover from an earlier iteration?